packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -622,6 +623,11 @@ module.exports = (
 
         await customs.check(request, primaryEmail, 'createEmail');
 
+        const account = await db.account(uid);
+        if (account.emails.length >= MAX_SECONDARY_EMAILS) {
```

Comment:
I would add a quick note here that the max means emails records (unverified + verified).